website/project/model.py

```diff
@@ -2299,6 +2311,7 @@ def register_node(self, schema, auth, data, parent=None):
         registered.save()
 
         if settings.ENABLE_ARCHIVER:
+            registered.reload()
```

Comment:
Without this, modularodm raises a `KeyExistsException` during the creation of a registration and fails.  For some reason, removing the backrefs necessitates reloading the registration and fork during creation.  I don't know enough about modularodm to know why.  I remember someone telling me that removing backrefs on some of the other items had encountered this same issue as well.